app/Jobs/AutomateSessionEntries.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Event;
+use App\Models\Session;
+use App\Models\Entry;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Carbon\Carbon;
+
+//Remove ShouldBeUnique if we partition the sessions to be handled
+class AutomateSessionEntries implements ShouldQueue, ShouldBeUnique
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        //Grab all existing live/past sessions in DB and load characters
+        $sessions = Session::where('start_time', '<=', Carbon::now())
+            ->where('start_time', '>', Carbon::now()->subMinutes(15))
+            ->with('characters')
+            ->get();
+
+        //entry creation input arrays for all characters & dm in $sessions
+        $input_arrays = collect([]);
```

Comment:
This variable isnt following naming conventions, it should be `$inputArrays`, camel case, not snake case :)

Reply:
True, brain still in python mode.